src/helpers/util.ts

```diff
@@ -8,7 +10,14 @@ export function runExternalProgram(
   programName: string,
   optionalArguments: string[] = [],
 ): string {
-  const result = childprocess.spawnSync(programName, optionalArguments)
+  const result = childprocess.spawnSync(
+    programName,
+    optionalArguments,
+    { maxBuffer: SPAWNPROCESSBUFFERSIZE },
+  )
+  if (result === undefined) {
+    throw new Error(`${programName}, ${typeof optionalArguments}, ${Array.isArray(optionalArguments)}, ${optionalArguments}`)
+  }
```

Comment:
What causes `result` to be `undefined`?

Reply:
@mitchell-codecov my latest change didn't push this file 😬. It was supposed to be removed